### ISSUE TITLE
setting page font: inheritance

### ISSUE THREAD

**Issue by rimmesbe** [jplisk03/prj-rev-bwfs-dasmoto#4]
Notice how you have to keep setting the font-family on each selector. It would be a lot easier to set this in one place, and we can with inheritance:
html { font-family: Helvetica; }
Now all elements will inherit this font from html.

https://github.com/jplisk03/prj-rev-bwfs-dasmoto/blob/master/Dasmoto's%20Arts%20%26%20Crafts/resources/style.css#L2

**Comment by jplisk03** [comment]
You would only do this if the whole page is be the same font correct?

**Comment by rimmesbe** [comment]
yes, or the majority of the site is one font, then you can set other fonts
more locally.
